packages/@lwc/integration-karma/scripts/karma-configs/test/sauce.js

```diff
@@ -56,14 +64,6 @@ const SAUCE_BROWSERS = [
         compat: true,
         nativeShadowCompatible: false,
     },
-    // TODO [#3083]: re-enable Firefox tests
-    // {
-    //     label: 'sl_firefox_compat',
-    //     browserName: 'firefox',
-    //     version: '54',
-    //     compat: true,
-    //     nativeShadowCompatible: false,
-    // },
```

Comment:
Removing the firefox compat tests as these require the geckodriver version = 0.30.0 but we are unable to specify the version using JWP format.

Since we plan to deprecate compat in 244, the chrome, ie ,and safari compat tests should be sufficient.

Reply:
If we can figure out a fix for Firefox 54 then great; otherwise yeah let's just not bother.